codespell_lib/data/dictionary.txt

```diff
@@ -15519,6 +15519,7 @@ interessted->interested
 interessting->interesting
 intereview->interview
 interfave->interface
+interfcae->interface
```

Comment:
Can you add the plural of this please. I know it's not your change but would you mind doing the same for the one above too.

Reply:
Done.